Cookbook/README.md

```diff
@@ -38,6 +38,7 @@ iOS Cookbook 👩‍🍳
 ##### ... learn functional programming 🚀
 
 * [Introduce "fish" operator (Kleisli composition)](./Proposals/Fish_Operator.md)
+* Learn about our functional operators [Coming soon...]
```

Comment:
Not sure if there's much value in adding "coming soon" item. I guess it helps to know that someone is working on this and direct any queries to you! 😄 I'd add "CNSMR-2693" there, so it's possible to check on the status.

Reply:
This is a public repo so I wouldn't add links to private JIRA Tickets. 
I believe we should only add this link once we have the documentation for it.